### DISCUSSION TITLE
Will AmberElec be ported to the new Arc-S?

### DISCUSSION THREAD

**Discussion by molul** [351ELEC/351ELEC#978] [Questions & Answers]
I'm a big AmberElec fan and I'm about to buy the new Anbernic Arc-S. Is there any hope for a port someday? The official UI isn't remotely as good.

**Comment by dhwz** [comment]
Yes maybe but it has to wait till we support the rk3566 platform.

**Reply by molul** [reply]
Thanks for the quick reply. Here's hoping this comes soon :)
